example_makefiles/makefile.inc.Linux

```diff
@@ -9,9 +9,11 @@
 # tested on CentOS 7, Ubuntu 16 and Ubuntu 14, see below to adjust flags to distribution.
 
 
-CC=g++
+CC=gcc
+CXX=g++
```

Comment:
Ok to replace CC with CXX.
I will replace CXX11 with CXX since c++11 is also required to compile the CPU lib now.